android/proguard-rules.pro

```diff
@@ -1,4 +1,9 @@
 -dontwarn com.amazon.**
 -keep class com.amazon.** {*;}
 -keepattributes *Annotation*
--keep class androidx.lifecycle.DefaultLifecycleObserver
\ No newline at end of file
+-keep class androidx.lifecycle.DefaultLifecycleObserver
+
+# The org.xmlpull package is part of the Android framework, so the classes are always available.
+# However, some apps add it to their classpath, either explicitly or transitively. When
+# this happens, it becomes susceptible to be shrunk. The following rule avoids that.
+-keep class org.xmlpull.v1.** { *; }
```

Comment:
Since we're adding it here, we should remove it from the ui package. It would be unnecessary there, considering this is a dependency of that package.